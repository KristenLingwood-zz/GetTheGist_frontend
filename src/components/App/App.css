@import 'pinacolada.css';
@import 'main.css';

