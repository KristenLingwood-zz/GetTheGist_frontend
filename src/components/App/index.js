import React, { Component } from 'react';
import Header from '../Header';
import Routes from '../Routes';
import Footer from '../Footer';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <Header />
        <Routes className="marginBottom-2 clearFix" />
        <Footer />
      </div>
    );
  }
}

export default App;
