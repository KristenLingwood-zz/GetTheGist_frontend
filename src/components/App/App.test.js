import React from 'react';
import ReactDOM from 'react-dom';
import App from '.';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <BrowserRouter>
      <App />
    </BrowserRouter>,
    div
  );
  ReactDOM.unmountComponentAtNode(div);
});

describe('<App />', () => {
  it('matches the snapshot', () => {
    var tree = renderer
      .create(
        <BrowserRouter>
          <App />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
