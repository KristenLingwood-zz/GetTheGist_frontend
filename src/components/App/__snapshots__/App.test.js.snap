// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches the snapshot 1`] = `
<div
  className="App"
>
  <nav
    className="navbar navbar-inverse navbar-fixed-top"
  >
    <div
      className="container-fluid"
    >
      <div
        className="navbar-header"
      >
        <a
          className="navbar-brand"
          href="/gists"
        >
          <i
            className="fas fa-cat"
          />
            
          GetTheGist
        </a>
        <button
          className="navbar-toggle"
          onClick={[Function]}
          type="button"
        >
          <span
            className="sr-only"
          >
            Toggle navigation
          </span>
          <span
            className="icon-bar"
          />
          <span
            className="icon-bar"
          />
          <span
            className="icon-bar"
          />
        </button>
      </div>
      <div
        aria-expanded={null}
        className="navbar-collapse collapse in"
      >
        <div
          className="navbar-form navbar-left"
        >
          <div
            className="form-group"
          >
            <input
              className="form-control"
              onChange={[Function]}
              placeholder="Search"
              type="text"
              value=""
            />
          </div>
          <button
            className="btn btn-default"
            disabled={false}
            onSubmit={[Function]}
            type="submit"
          >
            Submit
          </button>
        </div>
        <ul
          className="nav navbar-nav"
          role={null}
        >
          <li
            className=""
            role="presentation"
          >
            <a
              href="/gists"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              All gists
            </a>
          </li>
          <li
            className=""
            role="presentation"
          >
            <a
              href="https://github.com"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              GitHub
            </a>
          </li>
        </ul>
        <ul
          className="nav navbar-nav navbar-right"
          role={null}
        >
          <li
            className=""
            role="presentation"
          >
            <a
              href="/new"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              New gist
            </a>
          </li>
          <li
            className=""
            role="presentation"
          >
            <a
              href="#"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              User info
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
  <div
    className="container"
  >
    <h4
      id="loading_msg"
    >
      Getting the gist of it...
    </h4>
  </div>
  <div
    className="footer_links"
  >
    <div
      className="footer_left"
    >
      <p>
        <i
          className="far fa-copyright"
        />
         2018 Kristen Lingwood
      </p>
      <a
        href="http://kristenlingwood.com"
      >
        Terms
      </a>
      <a
        href="http://kristenlingwood.com"
      >
        Privacy
      </a>
      <a
        href="http://kristenlingwood.com"
      >
        Security
      </a>
      <a
        href="http://kristenlingwood.com"
      >
        Status
      </a>
      <a
        href="http://kristenlingwood.com"
      >
        Help
      </a>
    </div>
    <img
      alt="codercat"
      src="femalecodercat.jpg"
    />
    <div
      className="footer_right"
    >
      <a
        href="http://kristenlingwood.com"
      >
        Contact Kristen
      </a>
      <a
        href="http://kristenlingwood.com"
      >
        Pricing
      </a>
      <a
        href="http://kristenlingwood.com"
      >
        API
      </a>
      <a
        href="http://kristenlingwood.com"
      >
        Blog
      </a>
      <a
        href="http://kristenlingwood.com"
      >
        About
      </a>
    </div>
  </div>
</div>
`;
